target/linux/mediatek/dts/mt7981b-cudy-wr3000-v1.dts

```diff
@@ -41,37 +42,55 @@
 	leds {
 		compatible = "gpio-leds";
 
-		led_status: led@0 {
-			label = "blue:status";
-			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
-		};
-
-		led@1 {
-			label = "blue:internet";
-			gpios = <&pio 11 GPIO_ACTIVE_LOW>;
-		};
-
-		led@2 {
+		led-0 {
 			label = "blue:wan";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WAN;
+			function-enumerator = <0>;
 			gpios = <&pio 5 GPIO_ACTIVE_LOW>;
 		};
 
-		led@3 {
-			label = "blue:lan";
-			gpios = <&pio 9 GPIO_ACTIVE_LOW>;
-		};
-
-		led@4 {
+		led-1 {
 			label = "blue:wifi2";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WLAN;
+			function-enumerator = <0>;
```

Comment:
Maybe use `function-enumerator = <2>` (resp. `<5>`) signifying 2.4GHz (resp. 5GHz).
@chunkeey suggested this elsewhere.

Reply:
Another options are:
```
function-enumerator = <24>
function-enumerator = <50>
```

Reply:
I think they shoule be <0> & <1> to match the `radio0` & `radio1`.

Reply:
In any case this sounds like a project-wide decision rather than an mt7981 one.

Reply:
At the same time, it looks like "deprecated" labels are more understandable for the end user than modern enumerators and functions... And we still have to use a "salad" of labels (for the alias node) and new style at the same dts. I am increasingly inclined to think that now is not the time to unify anything. @raenye @raenye thank you for helping me realize this.

Reply:
At the same time, it looks like "deprecated" labels are more understandable for the end user than modern enumerators and functions... And we still have to use a "salad" of labels (for the alias node) and new style at the same dts. I am increasingly inclined to think that now is not the time to unify anything. @raenye @DragonBluep thank you for helping me realize this.

Reply:
> Another options are:
> 
> ```
> function-enumerator = <24>
> function-enumerator = <50>
> ```

![image](https://github.com/openwrt/openwrt/assets/82600916/de3f55f1-b68d-449f-ad90-8d34b46aab70)